enhancements/builders.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Removing Bare-metal Requirement For Builds
+authors:
+  - "@bcaton"
+reviewers:
+  - "@kleesc"
+  - "@bcaton"
+  - "@harishsurf"
+approvers:
+  - "@kleesc"
+  - "@bcaton"
+  - "@harishsurf"
+creation-date: 2021-12-03
+last-updated: 2021-12-03
+status: implementable
+---
+
+# remove-bare-metal-build-constraint
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+> 1. Will we encounter issues down the line running builds in unpriviliged containers with Podman?
+> 1. Does running builds in unprivileged containers give us the build isolation we need?
+
+## Summary
+
+Currently Quay builds run Podman commands in virtual machines launched via Pods. To run builds on virtual platforms this would require enabling nested virtualization, which is not possible in RHEL or OCP. This places the requirement of builds having to run on bare-metal clusters, which are a precious resource. This proposol is to remove the bare-metal constraint required to run builds.
```

Comment:
Mind calling out that this proposal is for an additional method of building images? 

> remove bare-metal constraint
^ This wording makes me wonder if bare-metal builds are going away.

Reply:
Updated.